pages/index.tsx

```diff
@@ -13,24 +13,20 @@ const Home: NextPage = () => {
     const { title, categories } = attributes as { title: string; categories: Array<Category> }
 
     return (
-        <div className={styles.container}>
+        <>
             <Head>
-                <title>RESPECT</title>
+                <title>Respect</title>
                 <meta name="description" content="Generated by create next app" />
                 <link rel="icon" href="/favicon.ico" />
             </Head>
 
-            <main className={styles.main}>
-                <div>
-                    <h1>{title}</h1>
-                    <HomeContent />
-                    Our services:
-                    <Categories categories={categories} />
-                </div>
+            <main className="u-content-width rich-text">
+                <h1>{title}</h1>
+                <HomeContent />
+                <p>Our services:</p>
+                <Categories categories={categories} />
             </main>
-
-            <footer className={styles.footer}></footer>
```

Comment:
Where did the footer go ? 😄

Reply:
it... was empty 😅 
my next PR will make a <Layout /> component, which will centralise the header and footer